app-demo/src/main/java/com/youappi/sdk/demo/YouAppApplication.java

```diff
@@ -17,7 +17,8 @@
   @Override
   public void onCreate() {
     super.onCreate();
-
-    YouAPPi.init(this, DEMO_TOKEN, true);
+    boolean consent = true;
```

Comment:
2 options:
* leave as it is and add "final"
* simply pass "true" directly to function param